src/createDispatcher.js

```diff
@@ -1,8 +1,12 @@
 import composeMiddleware from './utils/composeMiddleware';
 
+const INIT_ACTION = {
+  type: '@@INIT'
```

Comment:
Camelcase would be sexier ([example on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/iterator#Description))